contracts/switchboard/default-switchboards/SwitchboardBase.sol

```diff
@@ -40,6 +37,9 @@ abstract contract SwitchboardBase is ISwitchboard, AccessControlExtended {
     // destinationChainSlug => fees-struct with verificationFees and switchboardFees
     mapping(uint32 => Fees) public fees;
 
+    // destinationChainSlug => fees-struct with verificationFees and switchboardFees
+    mapping(uint32 => uint256) public initialPacketCount;
```

Comment:
destinationChainSlug is wrong wording, say sourceChainSlug or remoteChainSlug or siblingChainSlug